### PR TITLE
eventing

### PR DESCRIPTION
**Pull request by kamleshverma1**
eventing for clusters and nodes

Signed-off-by: kamlesh kaverma@redhat.com

### CODE REVIEW

File: app/components/requests/requests-controller.ts [hunk 1]
```diff
@@ -106,18 +107,22 @@ export class RequestsController {
...
+            host.task = result;
+            var callback = () => {
+                this.requestSvc.get(result).then((request) => {
+                    if (request.status === 'FAILED' || request.status === 'FAILURE') {
+                        this.$log.info('Failed to accept host in requests controller' + host.hostname);
+                        host.state = "FAILED";
+                        host.task = undefined;
+                    }
+                    else if (request.status === 'SUCCESS') {
+                        this.$log.info('Accepted host in requests controller ' + host.hostname);
```
Comment (kmkanagaraj) [inline app/components/requests/requests-controller.ts]: Are these changes related?

File: app/components/rest/server.ts [hunk 2]
```diff
@@ -24,10 +32,10 @@ export class ServerService {
     }
 
     // **getListByCluster**
-    // **@returns** a promise with all nodes part of the cluster.
+    // **@returns** a promise with all servers.
     getListByCluster(clusterId) {
-        return this.rest.one('clusters', clusterId).all('nodes').getList().then(function(nodes) {
-            return nodes;
+        return this.rest.one('clusters', clusterId).all('hosts').getList().then(function(servers) {
```
Comment (kmkanagaraj) [inline app/components/rest/server.ts]: Its 'nodes'. Why this is changed to 'hosts'?

File: app/components/requests/requests-controller.ts (hunk 1 above)
Comment (kamleshverma1) [inline app/components/requests/requests-controller.ts]: No , i did not updated here anything
